Formula/nift.rb

```diff
@@ -0,0 +1,19 @@
+class Nift < Formula
+  desc "Cross-platform open source framework for managing and generating websites"
+  homepage "https://nift.cc/"
+  url "https://github.com/nifty-site-manager/nsm/archive/v1.17.tar.gz"
+  sha256 "cdcc3b54d9d98c70a95e14396d09588017fdb2ecfcc821a96778251577b9bafe"
+
+  def install
+    system "make"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    Dir.mkdir "test"
+    Dir.chdir "test"
```

Comment:
Unnecessary. The test block already runs in a temporary directory

Reply:
`nsm init` needs to be run in an empty directory or an empty repo

Reply:
The changing directory to the parent at the end might be unnecessarily? I figured I'd leave it back where it started..

Reply:
Then:

```
mkdir "empty" do
  system "#{bin}/nsm", "init"
  assert_predicate testpath/"empty/site/index.html", :exist?
end
```

Reply:
Made the changes, tested them (including audit) and have pushed the changes. Let me know if I've missed anything.